### ISSUE TITLE
Add argument verbose to botscan() 

### ISSUE THREAD

**Issue by ryantmoore** [kurtawirth/botscan#14]
We currently use a loop in R to check multiple handles for botiness.  We should add an argument for verbose printing in `botscan()`.

1. Add argument `verbose = TRUE`
2. Document it
3. At the top of the loop (line 57), add 
```
if(user_idx %% 10 == 0){
  cat(paste("Checking user number", user_idx, "\n"))
}

**Comment by ryantmoore** [comment]
Initial version included in commit c7668fb.

**Comment by kurtawirth** [comment]
This is tested and working properly. Closing issue.
